src/installer/pkg/sfx/Microsoft.NETCore.App/PackageOverrides.txt

```diff
@@ -132,33 +154,39 @@ System.Runtime.Numerics|4.3.0
 System.Runtime.Serialization.Formatters|4.3.0
 System.Runtime.Serialization.Json|4.3.0
 System.Runtime.Serialization.Primitives|4.3.0
-System.Security.AccessControl|4.4.0
+System.Runtime.Serialization.Xml|4.3.0
+System.Security.AccessControl|6.0.1
 System.Security.Claims|4.3.0
-System.Security.Cryptography.Algorithms|4.3.0
-System.Security.Cryptography.Cng|4.4.0
+System.Security.Cryptography.Algorithms|4.3.1
+System.Security.Cryptography.Cng|5.0.0
 System.Security.Cryptography.Csp|4.3.0
 System.Security.Cryptography.Encoding|4.3.0
-System.Security.Cryptography.OpenSsl|4.4.0
+System.Security.Cryptography.OpenSsl|5.0.0
 System.Security.Cryptography.Primitives|4.3.0
-System.Security.Cryptography.X509Certificates|4.3.0
-System.Security.Cryptography.Xml|4.4.0
+System.Security.Cryptography.X509Certificates|4.3.2
 System.Security.Principal|4.3.0
-System.Security.Principal.Windows|4.4.0
+System.Security.Principal.Windows|5.0.0
+System.Security.SecureString|4.3.0
 System.Text.Encoding|4.3.0
+System.Text.Encoding.CodePages|9.0.0
 System.Text.Encoding.Extensions|4.3.0
-System.Text.RegularExpressions|4.3.0
+System.Text.Encodings.Web|9.0.0
+System.Text.Json|9.0.0
+System.Text.RegularExpressions|4.3.1
 System.Threading|4.3.0
+System.Threading.Channels|9.0.0
 System.Threading.Overlapped|4.3.0
 System.Threading.Tasks|4.3.0
-System.Threading.Tasks.Extensions|4.3.0
+System.Threading.Tasks.Dataflow|9.0.0
+System.Threading.Tasks.Extensions|5.0.0
 System.Threading.Tasks.Parallel|4.3.0
 System.Threading.Thread|4.3.0
 System.Threading.ThreadPool|4.3.0
 System.Threading.Timer|4.3.0
-System.ValueTuple|4.3.0
-System.Xml.ReaderWriter|4.3.0
+System.ValueTuple|4.5.0
+System.Xml.ReaderWriter|4.3.1
 System.Xml.XDocument|4.3.0
 System.Xml.XmlDocument|4.3.0
 System.Xml.XmlSerializer|4.3.0
 System.Xml.XPath|4.3.0
-System.Xml.XPath.XDocument|4.3.0
+System.Xml.XPath.XDocument|5.0.0
```

Comment:
We haven't shipped a new version of `System.Xml.XPath.XDocument` since 2016. Why does it get `5.0.0` and the other System.Xml packages don't?

Reply:
This package and a few others (i.e. DispatchProxy that you commented on below) will soon get a new version for servicing. The sources got moved to https://github.com/dotnet/maintenance-packages/tree/main/src.